### PR TITLE
Api authorisation

### LINKED ISSUE #4: Create model User
Create app authentication and model User

### PR DESCRIPTION
**Pull request by Space4sheep**

### CODE REVIEW

File: Backend/test_project/test_project/settings.py [hunk 1]
```diff
@@ -9,9 +9,11 @@
 For the full list of settings and their values, see
 https://docs.djangoproject.com/en/4.2/ref/settings/
```
Comment (llpvqll) [inline Backend/test_project/test_project/settings.py]: remove this text

File: Backend/test_project/test_project/settings.py [hunk 2]
```diff
@@ -80,10 +90,22 @@
     }
 }
 
+# Email settings
+# Email account: awesomeshopsecondteam@gmail.com
+# Password: emptypassword123
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = 'smtp.gmail.com'
```
Comment (llpvqll) [inline Backend/test_project/test_project/settings.py]: you can add .env file, and here you can make as os.getenv("KEY_FOR_EVERYONE_ENVIRONMENT_VARIABLE")

Comment (llpvqll) [reply Backend/test_project/test_project/settings.py]: forgot about 
from dotenv import load_dotenv
load_dotenv()

Comment (Space4sheep) [reply Backend/test_project/test_project/settings.py:97]: image
Like this?

File: Backend/test_project/test_project/settings.py (hunk 1 above)
Comment (Space4sheep) [reply Backend/test_project/test_project/settings.py:10]: +

File: Backend/test_project/test_project/settings.py (hunk 2 above)
Comment (Space4sheep) [reply Backend/test_project/test_project/settings.py]: image
Thank you for the order. I did it like this. Everything works. I just wonder how other developers can get the data from .env, I'm not going to commit this file.

Comment (llpvqll) [reply Backend/test_project/test_project/settings.py]: you can add .env.example file to the github, with all constants but without credentials. everyone who clone repository can create .env file, copy constants from .env.example and set herself credentials

**Review by llpvqll — APPROVED** [review]

**Review by llpvqll — APPROVED** [review]

**Review by antony-kosenko — APPROVED** [review]

**Review by lolesia — APPROVED** [review]

**Pull request merged**